web/pageComponents/topicPages/FullWidthVideo.tsx

```diff
@@ -14,20 +14,16 @@ const StyledFigure = styled.figure`
   background-color: transparent;
 `
 
-interface StyledHLSPlayerProps {
-  aspectRatio: VideoPlayerRatios
-}
-
-const StyledHLSPlayer = styled(HLSPlayer)<StyledHLSPlayerProps>`
+const StyledHLSPlayer = styled(HLSPlayer)<{ $aspectRatio?: FullWidthVideoRatio }>`
   object-fit: cover;
   width: 100%;
-  height: ${(props) => {
-    switch (props.aspectRatio) {
-      case VideoPlayerRatios.narrow:
-        return '40vh'
-      case VideoPlayerRatios.fullScreen:
+  height: ${({ $aspectRatio }) => {
+    switch ($aspectRatio) {
+      case 'narrow':
+        return '30vh'
```

Comment:
When I mentioned looking at how the full-width image was implemented was in regards to the proportion of the banners, since there had been some back-and-forth adjustments to make it work for all resolutions.

With `30vh`, we run into these types of issues where the video is way too narrow since it uses the screen proportion and not the video:
![image](https://github.com/equinor/energyvision/assets/23337266/a39fbdb2-c390-4889-9dff-a93ee758ca67)

The way the full-screen narrow hero is implemented is by using 4:3 for small screens and 10:3 for large screens, we should follow the same logic for videos.

It can be a bit tricky to implement video ratios, but always useful to learn new tricks :P try to search for video ratios with padding-bottom. If you get stuck ping me on slack.